src/github-helper.ts

```diff
@@ -42,18 +42,18 @@ export async function createPullRequest(
     const title =
       github.context.payload &&
       github.context.payload.pull_request &&
-      github.context.payload.pull_request.title
-    core.info(`Using body '${title}'`)
+      (github.context.payload.pull_request.title as unknown as string)
+    core.info(`Using title '${title || ''}'`)
```

Comment:
```suggestion
    core.info(`Using title '${title ?? ''}'`)
```

Null coalescing operator is preferred here?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing